src/commands/app/create.ts

```diff
@@ -66,8 +56,12 @@ export default class AppCreate extends Command {
         debug(JSON.stringify(error.response.data));
       }
 
+      if (error.response && error.response.status === 404) {
+        this.error(`Could not create the app.`);
+      }
+
       if (error.response && error.response.status === 409) {
-        this.error(`The app already exists. Please use a unique name for your app.`);
+        this.error(`Another operation is already running. Please wait.`);
```

Comment:
Why this has been changed? Have you rebased this branch from master?

Reply:
I'm sure I did it...
It seems when I was resolving conflict choose the wrong change...(my mistake)
If it is possible I'll make this log error back in the new commit.

Reply:
Yes please revert the error message to the previous one.